api/v1alpha1/flowcollector_webhook.go

```diff
@@ -53,6 +53,10 @@ func (r *FlowCollector) ConvertTo(dstRaw conversion.Hub) error {
 		dst.Spec.Processor.ConnectionEndTimeout = restored.Spec.Processor.ConnectionEndTimeout
 	}
 
+	if restored.Spec.Processor.Metrics.DisableAlerts != nil {
```

Comment:
out of curiosity, did you have "something" telling you that you had something to do here, or was it something you had to keep in mind yourself? (trying to understand how error prone it can be if we forget about the conversion)

Reply:
After adding the new field in v1beta1 running make generate was triggering some warning because of the new fields.

But then I mostly looked at previous PR to understand how to do it.